### ISSUE TITLE
Automatically generated content

### ISSUE THREAD

**Issue by kulla** [serlo-org/athene2-rfcs#17]
I would like to have plugins / new elements which automatically can generate content while generating the article. Lets imagine that `<filter>...</filer>` defines such an element. The element `<plot>sin(x^2)</plot>` might create a plot of the function `sin(x^2)` at the place were it is defined. There might be also an element

```
<solve>
x+y=5
x-y=2
</solve>
```

would create a full solution of the system of linear equations `x+y=5` and `x-y=2`.

**Comment by kulla** [comment]
There should be a prototype first before we start to invest big effort in this issue.

**Comment by kulla** [comment]
Duplicate of #47
